test/integration/api_authentication_test.rb

```diff
@@ -5,7 +5,8 @@ class ApiAuthenticationTest < ActionDispatch::IntegrationTest
   include TokenAuthSupport
 
   def access_user_endpoint(token = nil, params = {})
-    get "/user.json", params, token.nil? ? {} : {"HTTP_AUTHORIZATION" => "Bearer #{token}"}
+    headers = token.nil? ? {} : {"HTTP_AUTHORIZATION" => "Bearer #{token}"}
+    get "/user.json", params: params, headers: headers
```

Comment:
Prefer this approach, but is this related to the upgrade?

Reply:
Headers had to be passed in as a hash - didn't think youd be allowed:

```get "user.json", params: params, headers: token.nil? {} : {"HTTP...etcetc"}```